spec/index.bs

```diff
@@ -632,6 +653,26 @@ dictionary GPUBufferDescriptor : GPUObjectDescriptorBase {
         </div>
 </dl>
 
+## Buffer Destruction ## {#buffer-destruction}
+
+An application that doesn't need a {{GPUBuffer}} can choose to lose access to
+it before garbage collection by calling {{GPUBuffer/destroy()}}. Non normatively
```

Comment:
I wonder if we need a section explaining how JS garbage collection affects us (to refer from here), or to link to an external documentation source that does that

Reply:
Is that how other standards do it? If the knowledge of GC is implied, then it's cool. Otherwise, we might need a non-normative paragraph explaining what GC does.

Reply:
I'm not sure, @beaufortfrancois do you know about this?

Reply:
From my experience, GC is implied.